introduction.md

```diff
@@ -4,3 +4,4 @@ created new branch and pushed to git
 * git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?